code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -507,41 +507,48 @@
 		icon_state = initial(icon_state)
 
 /obj/structure/dropship_equipment/electronics/landing_zone_detector
+
 	name = "\improper AN/AVD-60 LZ detector"
-	desc = "An electronic device linked to the dropship's camera system that lets you observe your landing zone mid-flight."
+	desc = "An electronic device linked to the dropship's camera system that lets you observe your landing zone."
+
 	icon_state = "lz_detector"
 	point_cost = 50
 	var/obj/structure/machinery/computer/cameras/dropship/linked_cam_console
 
+/obj/structure/dropship_equipment/electronics/landing_zone_detector/proc/connect_cameras()
+	if(linked_cam_console)
+		return
+	for(var/obj/structure/machinery/computer/cameras/dropship/dropship_camera_console in range(5, loc))
+		linked_cam_console = dropship_camera_console
+		break
+	linked_cam_console.network.Add(CAMERA_NET_LANDING_ZONES)
+
+/obj/structure/dropship_equipment/electronics/landing_zone_detector/proc/disconnect_cameras()
+	if(!linked_cam_console)
+		return
+	linked_cam_console.network.Remove(CAMERA_NET_LANDING_ZONES)
+	for(var/datum/weakref/ref in linked_cam_console.concurrent_users)
```

Comment:
as anything

Reply:
for(var/datum/weakref/ref as anything in linked_cam_console.concurrent_users)